PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -284,6 +284,9 @@ def miniAOD_customizeCommon(process):
     ## Force a re-run of the tau id during MiniAOD production stage
     process.load('RecoTauTag.Configuration.RecoPFTauTag_cff')
 
+    #bunch spacing producer
+    process.load('RecoLuminosity.LumiProducer.bunchSpacingProducer_cfi')
```

Comment:
> On Sep 29, 2015, at 1:56 PM, Lindsey Gray <notifications@github.com> wrote:
> 
> In PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py:
> 
> > @@ -284,6 +284,9 @@ def miniAOD_customizeCommon(process):
> >      ## Force a re-run of the tau id during MiniAOD production stage
> >      process.load('RecoTauTag.Configuration.RecoPFTauTag_cff')
> >  
> > +    #bunch spacing producer
> > +    process.load('RecoLuminosity.LumiProducer.bunchSpacingProducer_cfi')
> 
> MiniAOD is run in unscheduled mode, there are no sequences.
> 

right - but this is a fairly high level config (and miniAOD specific). 

maybe better in Configuration/StandardSequences/PAT_cff

but still thinking..


> —
> Reply to this email directly or view it on GitHub.
>